src/Resources/skeleton/validator/Validator.tpl.php

```diff
@@ -9,7 +9,7 @@ class <?= $class_name ?> extends ConstraintValidator
 {
     public function validate($value, Constraint $constraint)
     {
-        /* @var $constraint \<?= $constraint_class_name ?> */
+        /** @var \<?= $constraint_class_name ?> $constraint */
```

Comment:
```suggestion
        /** @var <?= $constraint_class_name ?> $constraint */
```

Reply:
Even though this does give a small perf boost - its not consistent with the rest of the generated code in maker bundle

Reply:
It was already there I just changed the order here

Reply:
Correct, but CI would flag this as an issue if it could "see" the actual class being declared. We only use the root name space for a select list of PHP functions.